src/components/i18n/i18n.stories.mdx

```diff
@@ -0,0 +1,75 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import i18n from 'i18n-js';
+import I18nComponent from './i18n.js';
+import DeprecationWarning from '../../__internal__/DeprecationWarning';
+
+<Meta title="i18nComponent" parameters={{ info: { disable: true }}} />
+
+<DeprecationWarning>
+  It is on our roadmap to remove our I18n implementation
+  and provide localisation for our components out of the box.
+  If you are adding localisation to your application we recommend you use a dedicated solution such as react-i18next
+</DeprecationWarning>
+
+# I18nComponent (Internationalization)
+Displays a translation from an i18n file.
+
+Internationalization (i18n) means apps can be localized for languages and cultures easily.
+This component displays a single text translation.
+You can use this component with any other component that displays text.
+
+## Quick Start
+Import `I18nComponent` into the project.
+
+```jsx
+import I18n from "carbon-react/lib/components/i18n";
+
+const MyComponent = () => (
+  <I18nComponent scope='Foo'/>
+);
+```
+
+### Default
+<Preview>
+  <Story name="Default">
+    {() => {
+        i18n.translations.en['my'] = {
+        example: '# My __example__ translation.'
+      }; 
+      return (
+        <I18nComponent scope='my.example'/>
+      )         
+    }}
+  </Story>
+</Preview>
+
+### Markdown
+<Preview>
+  <Story name="with Markdown">
+    {() => {
+        i18n.translations.en['my'] = {
+        example: '# My __example__ translation.'
+      }; 
+      return (
+        <I18nComponent markdown scope='my.example'/>
+      )         
+    }}
+  </Story>
+</Preview>
+
+### Inline
+<Preview>
```

Comment:
there is no change between `default` and `with Inline` so we need to ignore `chromatic` for this story

Reply:
I've added the parameter to the story to prevent a chromatic snapshot being taken 😄